src/api/torii-endpoints.md

```diff
@@ -0,0 +1,332 @@
+# TYPE tx_amount histogram
+tx_amount_bucket{le="0.005"} 0
+tx_amount_bucket{le="0.01"} 0
+tx_amount_bucket{le="0.025"} 0
+tx_amount_bucket{le="0.05"} 0
+tx_amount_bucket{le="0.1"} 0
+tx_amount_bucket{le="0.25"} 0
+tx_amount_bucket{le="0.5"} 0
+tx_amount_bucket{le="1"} 0
+tx_amount_bucket{le="2.5"} 0
+tx_amount_bucket{le="5"} 0
+tx_amount_bucket{le="10"} 0
+tx_amount_bucket{le="+Inf"} 2
+tx_amount_sum 26
+tx_amount_count 2
+# HELP txs Transactions committed
+# TYPE txs counter
+txs{type="accepted"} 1
+txs{type="rejected"} 0
+txs{type="total"} 1
+# HELP uptime_since_genesis_ms Network up-time, from creation of the genesis block
+# TYPE uptime_since_genesis_ms gauge
+uptime_since_genesis_ms 54572974
+# HELP view_changes Number of view_changes in the current round
+# TYPE view_changes gauge
+view_changes 0
+```
+
+:::
+
+Learn [how to use metrics](/guide/advanced/metrics).
+
+## Query
+
+- **Protocol:** HTTP
+- **Method:** `POST`
+- **Endpoint:** `/query`
+- **Expects:**
+  - **Body:** SCALE-encoded[^1] [`VersionedSignedQuery`](/api/data-model-schema#versionedsignedquery)
+  - **Query parameters:**
+    - **`start`:** An optional parameter in queries where results can be indexed. Use to return results from a specified
+      point. Results are ordered by id, which uses Rust's
+      [PartialOrd](https://doc.rust-lang.org/std/cmp/trait.PartialOrd.html#derivable) and
+      [Ord](https://doc.rust-lang.org/std/cmp/trait.Ord.html) traits.
+    - **`limit`:** An optional parameter in queries where results can be indexed. Use to return a specific number of
+      results.
+    - **`sort_by_metadata_key`:** An optional parameter in queries. Use to sort results containing metadata with a given
+      key.
+
+**Responses:**
+
+| Response           | Status | Body                                                                                       |
+| ------------------ | ------ | ------------------------------------------------------------------------------------------ |
+| Success            | 200    | [`VersionedBatchedResponse<Value>`](/api/data-model-schema#versionedbatchedresponse-value) |
+| Conversion Error   | 400    | [`QueryExecutionFail::Conversion(String)`](/api/data-model-schema#queryexecutionfail)      |
+| Evaluate Error     | 400    | [`QueryExecutionFail::Evaluate(String)`](/api/data-model-schema#queryexecutionfail)        |
+| Unauthorized Error | 401    | [`QueryExecutionFail::Unauthorized`](/api/data-model-schema#queryexecutionfail)            |
+| Signature Error    | 401    | [`QueryExecutionFail::Signature(String)`](/api/data-model-schema#queryexecutionfail)       |
+| Permission Error   | 403    | [`QueryExecutionFail::Permission(String)`](/api/data-model-schema#queryexecutionfail)      |
+| Find Error         | 404    | [`QueryExecutionFail::Find(FindError)`](/api/data-model-schema#queryexecutionfail)         |
+
+TODO: ask the team about unauth error vs account not found
```

Comment:
I am not sure about how `Unauthorized` works. If the client makes a query using an account which is not found, will it be `QueryExecutionFail::Unauthorized` or `QueryExecutionFail::Find(FindError::Account(_))`?

Reply:
More context:

- https://github.com/hyperledger/iroha/issues/3951

It should be deleted from the doc.